eZ/Publish/API/Repository/SectionService.php

```diff
@@ -57,9 +57,7 @@ public function updateSection(Section $section, SectionUpdateStruct $sectionUpda
     public function loadSection($sectionId);
 
     /**
-     * Loads all sections.
-     *
-     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException If the current user user is not allowed to read a section
+     * Loads all sections. If the current user is not allowed to read a section then list are filtered.
```

Comment:
Maybe something like:
```suggestion
     * Loads all sections, excluding the ones the current user is not allowed to read.
```